packages/api-v4/src/images/types.ts

```diff
@@ -18,6 +20,19 @@ export interface ImageRegion {
   status: ImageRegionStatus;
 }
 
+export interface ImageSharingData {
+  shared_by: null | {
+    sharegroup_id: number;
+    sharegroup_label: string;
+    sharegroup_uuid: string;
+    source_image_id: number;
+  };
+  shared_with: null | {
+    shared_count: number;
```

Comment:
```suggestion
    sharegroup_count: number;
```

I think this should be `sharegroup_count`